DoSomethingArchive/gambit-content
# Removes legacy /reportback endpoint

#### What's this PR do?
* Removes the `/reportback/:campaignName` endpoint, now deprecated in favor of using a single Chatbot Mobile Commons Campaign and mData.
* Removes unused functions from `lib/helpers`
* Removes unused packages from `package.json`

#### How should this be reviewed?
* Verify entire CampaignBot and DonorsChooseBot conversations work as expected
* Verify `/reportback/:campaignName` endpoints return a 404. This will likely cause a few more Mobile Commons emails for 404's until we de-activate all the mData's that we no longer use.

#### Any background context you want to provide?
Last item left to do here is add the entire `lib` directory to `scripts.lint` in our `package.json` and lint the legacy `lib/helpers.js` 

#### Relevant tickets
Very big dent in #706

#### Checklist
